doc/writing.md

```diff
@@ -150,14 +148,14 @@ Hello. ++I have added this part++
 
 ### Line break
 
-You can force line breaks in text (carriage return) using `<br>`, which is the
+You can force line breaks in text (carriage return) using `<br>`, similar to
```

Comment:
I'll leave that up to you, I think your change would be fine. I'm really nitpicking here! 😄